src/subdomains/lightning/controllers/lightning-transaction.controller.ts

```diff
@@ -0,0 +1,15 @@
+import { Controller, Get, Query } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { LightningTransactionDto } from '../dto/lightning-transaction.dts';
+import { LightningTransactionService } from '../services/lightning-transaction.service';
+
+@ApiTags('Transaction')
+@Controller('info')
+export class LightingTransactionController {
+  constructor(private lightningTransactionService: LightningTransactionService) {}
+
+  @Get('getTransactionInfo')
+  async getTransactionInfo(@Query('id') id: string): Promise<LightningTransactionDto[]> {
+    return this.lightningTransactionService.getTransactionInfo(id);
+  }

```

Comment:
This API should only return a single transaction

Reply:
In Case of a Routing there are two Transactions - a "input" and a "output".

Reply:
Good point ... we should ignore routing transactions for this API -> filter out during SQL query.